src/components/plugin/option.js

```diff
@@ -48,13 +51,26 @@ class PluginOption extends PureComponent {
       case 'bool':
       case 'boolean':
         return <FormToggle {...this.attrs}/>
+      case 'template':
+        return (
+          <FormElement id={this.props.spec.label} isCompact>
+            <TemplateSelect {...this.attrs}
+              isRequired
```

Comment:
`isRequired` is one of the properties that the plugin configuration determines. We can't override it here!